assets/js/modules/analytics/datastore/report.js

```diff
@@ -32,11 +33,13 @@ import { stringifyObject } from '../../../util';
 import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
 import { isValidDateRange, isValidOrders } from '../../../util/report-validation';
 import { isValidDimensions, isValidMetrics } from '../util/report-validation';
+import { actions as adsenseActions } from './adsense';
+import { normalizeReportOptions } from '../util/report-normalization';
 
 const fetchGetReportStore = createFetchStore( {
 	baseName: 'getReport',
 	controlCallback: ( { options } ) => {
-		return API.get( 'modules', 'analytics', 'report', options );
+		return API.get( 'modules', 'analytics', 'report', normalizeReportOptions( options ) );
```

Comment:
Normalization is applied late here rather than in `argsToParams` (as I initially thought) to not interfere with our calls to `receiveGetReport` which takes `params`.